tests/providers/amazon/aws/operators/test_ecs_system.py

```diff
@@ -0,0 +1,96 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+
+from tests.test_utils.amazon_system_helpers import AWS_DAG_FOLDER, AmazonSystemTest
+
+
```

Comment:
```suggestion
@pytest.mark.backend("mysql", "postgres")
```
See also: https://github.com/apache/airflow/blob/master/TESTING.rst#running-backend-specific-tests

Actually I think I do not quite understand the backend marker myself. I am wondering why for example do we use `@pytest.mark.backend("mysql", "postgres")` [here](https://github.com/apache/airflow/blob/b6f919cbc5f6784ee119da8f7a34348514f09d70/tests/providers/google/cloud/operators/test_automl_system.py#L24). What does this example dag have against running on `sqlite`? Why do we need the marker here?

Maybe @turbaszek knows more about it :)

Reply:
you are right it should work but I didn't test it with `sqlite`

:heavy_check_mark: done, I have added mark-backend